### PR TITLE
fix: account addresses separator

### PR DESCRIPTION
**Pull request by alter-eggo**
> Try out Leather build 342594f — [Extension build](https://github.com/leather-io/extension/actions/runs/12480995682), [Test report](https://leather-io.github.io/playwright-reports/fix-account-sep), [Storybook](https://fix-account-sep--65982789c7e2278518f189e7.chromatic.com), [Chromatic](https://www.chromatic.com/library?appId=65982789c7e2278518f189e7&branch=fix-account-sep)

Fixes account addresses bullet separator

was
Screenshot 2024-12-23 at 19 38 34

now
Screenshot 2024-12-23 at 19 38 50

### CODE REVIEW

File: src/app/components/loaders/bitcoin-account-loader.tsx [hunk 1]
```diff
@@ -20,11 +21,9 @@ interface BtcAccountLoaderIndexProps extends BitcoinAccountLoaderBaseProps {
 
 type BtcAccountLoaderProps = BtcAccountLoaderCurrentProps | BtcAccountLoaderIndexProps;
 
-export function BitcoinNativeSegwitAccountLoader({
-  children,
-  fallback,
-  ...props
-}: BtcAccountLoaderProps) {
+export function useBitcoinNativeSegwitAccountLoader(
+  props: DistributiveOmit<BtcAccountLoaderProps, 'children' | 'fallback'>
```
Comment (alter-eggo) [inline src/app/components/loaders/bitcoin-account-loader.tsx]: https://stackoverflow.com/questions/57103834/typescript-omit-a-property-from-all-interfaces-in-a-union-but-keep-the-union-s

maybe we can add https://www.npmjs.com/package/type-fest ?

Comment (edgarkhanzadian) [reply src/app/components/loaders/bitcoin-account-loader.tsx]: Hmm since like an interesting package, i'm up for it

**Review by edgarkhanzadian — APPROVED** [review]
🚀

File: src/app/components/loaders/bitcoin-account-loader.tsx (hunk 1 above)
Comment (alter-eggo) [reply src/app/components/loaders/bitcoin-account-loader.tsx]: added

**Pull request merged**